src/screens/DecideStackScreen.js

```diff
@@ -18,7 +18,6 @@ export function DecideStackScreen({ navigation }) {
   const dispatch = useDispatch();
 
   useEffect(() => {
-    dispatch(resetData());
```

Comment:
I couldn't find an explanation for why this was here, maybe something will break on the QA, but I did some tests and couldn't find anything wrong

Reply:
This was setting the redux store to `INITIAL_STATE` on the `DecideStackScreen`, which is the first screen the app goes to decide if it already has a loaded wallet or should display the new wallet screen

I have absolutely no idea why this was being done, I've checked and the state at this point was already the initial state